.circleci/publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eu
+
+echo $GPG_KEY | base64 --decode | gpg --batch --import --passphrase "$GPG_PASSPHRASE"
+
+gpg --passphrase $GPG_PASSPHRASE --batch --yes -a -b LICENSE
+
+mill -i src.io.iohk.decco.publish \
+--sonatypeCreds "$OSS_USERNAME:$OSS_PASSWORD"  \
+--gpgPassphrase "$GPG_PASSPHRASE" \
+--gpgKeyName "$GPG_KEY_ID" \
+--release false
+
+mill -i src.io.iohk.decco.auto.publish \
```

Comment:
Have you tried `mill -i __.publish ...` for publishing all modules in a single call?

Reply:
That will release other modules like test etc as well I am not sure we need that to be released

Reply:
In fact, it does release the tests while I expect mill not to do it, thanks.